examples/src/main/java/alluxio/cli/TestRunner.java

```diff
@@ -155,4 +160,14 @@ private static int runTest(OperationType opType, ReadType readType, WriteType wr
     }
     return result ? 0 : 1;
   }
+
+  /**
+   * @return working directory for the test suite
+   */
+  private String getTestPath() {
+    if (mDirectory == null) {
```

Comment:
that will through an error in case the directory is a mount point as we try to delete it